tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -335,9 +326,6 @@ def setup_client_security(self,
 
     def cleanup(self, *, force=False):
         # Cleanup in the reverse order of creation
-        # TODO(sergiitk): remove next line once proxy deletion is not dependent
-        # upon proxy type.
-        self.target_proxy_is_http = True
```

Comment:
We can get rid of the flag `target_proxy_is_http` altogether right? Since security also now support grpc proxy ...

Reply:
I think we should keep the option of using HTTP target proxy, just in case. There's a TODO to get rid of the flag though:
https://github.com/grpc/grpc/blob/657b7c49de07e1d4462d15555266a57bf7cf0916/tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py#L69-L70

Reply:
Sounds good